### PR TITLE
Add DVR minimum size to Options

### PR DESCRIPTION
**Pull request by geisonfgf**
### Goal
Enable minDvrTime be set by options

### How to test
1 - In Example app set the option[kMinDvrOption] to 15
2 - Run the app and check if is possible make a seek in a Live video
3 - Run all tests

### CODE REVIEW

File: Sources/Clappr_iOS/Classes/Base/Options.swift [hunk 1]
```diff
@@ -11,6 +11,7 @@ public let kPlaybackNotSupportedMessage = "playbackNotSupportedMessage"
 public let kMimeType = "mimeType"
 public let kDefaultSubtitle = "defaultSubtitle"
 public let kDefaultAudioSource = "defaultAudioSource"
+public let kMinDvrOption = "minDvrTime"
```
Comment (igoriols) [inline Sources/Clappr_iOS/Classes/Base/Options.swift]: Hey @geisonfgf , I suggest change the name to `kMinDvrTime` to make it self explaining.

Comment (dezouzart) [reply Sources/Clappr_iOS/Classes/Base/Options.swift]: web is using this name `minDvrSize`

**Comment by dezouzart** [comment]
@geisonfgf , should we update the docs to include this new option?

**Comment by geisonfgf** [comment]
@dezouzart I had understood that this option is only for tests so it should not be available in the docs.

File: Sources/Clappr_iOS/Classes/Base/Options.swift (hunk 1 above)
Comment (nabilsafatli) [reply Sources/Clappr_iOS/Classes/Base/Options.swift]: Hey guys, let's use the Player Web name.

**Comment by nabilsafatli** [comment]
Hey guys,
The PR name is `feat: enable minDvrTime be set by options`

**Comment by nabilsafatli** [comment]
Hey @geisonfgf ,
Each client player has its needs and may expect a different minimum size for DVR.
Even though it was for testing reasons, as it is available here it should be documented. 😄

**Comment by igoriols** [comment]
@nabilsafatli , we added `kMinDvrSize` to the [options list](https://github.com/clappr/clappr-ios/wiki/Options).

**Pull request merged**